### ISSUE TITLE
Time is wrong

### ISSUE THREAD

**Issue by dantveita** [soberstadt/simple-clock-widget#3]
All of a sudden, the time (hour) is not the same as system time. How is this possible? And how would i change it? Seems to be related to timezone, becuase minutes are synced correctly. But i see no way of changing this in the *.coffe file.

**Comment by soberstadt** [comment]
This doesn't seem possible. Can you open a terminal window and run `date` and see if it is the system time you expect to see?

**Comment by dantveita** [comment]
I know this is very wierd. Take a look: imagizer.imageshack.com/img907/9687/l1kZIb.png

(Timezone is correct, systemtime is correct. Ubersicht is two hours ahead.)

**Comment by dantveita** [comment]
Oops deleted the previous comment by mistake. Meant to edit it. Here goes again: screenshot of my setup: http://imagizer.imageshack.com/img907/9687/l1kZIb.png

Everything is correct except ubersicht which is two hours ahead.

**Comment by soberstadt** [comment]
Wow, that is super weird. What about `date +"%d/%m/%Y %a\n%l:%M %p"`?

**Comment by dantveita** [comment]
Same thing, only 12-hour instead of 24-hour.

**Comment by soberstadt** [comment]
I don't know. I'm assuming you have tried restarting your computer. @felixhageloh any ideas?

**Comment by felixhageloh** [comment]
not sure ... from the screenshots it looks like you are using Z Shell. @dantveita could you maybe check running `data` inside a bash shell and see what the output is there (Übersicht uses bash)

**Comment by dantveita** [comment]
same thing in bash http://imagizer.imageshack.com/img908/4024/8l7FH1.png

**Comment by dantveita** [comment]
actually... i didnt pay attention before previous post. The hour has not changed. It should be 17 now shouldnt it? Need to check the .coffee file again....

**Comment by felixhageloh** [comment]
I see .. maybe check the debug console for errors. It sounds like the widget has just stopped updating

**Comment by dantveita** [comment]
This is so unbeliably wierd and awkward, but the time-format was all wrong. The 'hour' was set to %y, not %H. That is dumb in and of itself. But the wierd thing is that either I must have made this mistake to begin with - which means i have had the wrong time on my desktop for close to a year which does not make any sense at all. So it must have been changed somehow - but i have no idea how that happened.

I couldnt have accidentally set the hour to show the year (15) when I first configured it - at 3pm? 

Anyway.. sorry to bother you about this blunder. Its all good now.
